src/components/home/Post.tsx

```diff
@@ -0,0 +1,177 @@
+/** @jsx jsx */
+import React, { useEffect, useRef, useState } from 'react';
+import {jsx, SxStyleProp} from 'theme-ui';
+import {AiOutlineExpandAlt} from 'react-icons/ai';
+import { SeeMoreButton } from './SeeMoreButton';
+
+export interface IPost {
```

Comment:
Should this interface be here? Will there be another use for this interface outside this component, because I think something like `import components.home.Post.IPost` feels a bit strange.

Reply:
where do you suggest it to be, because originally I was thinking there can be a centralized place (`interface.ts`) but importing different components from one file also feels weird. Like:  
```ts
import { Post } from 'components/home/Post';
import { IPost } from 'utils/interface';
```

Reply:
I feel like I've certainly seen a centralized file with multiple loosely-related classes and functions thrown together just to keep it neat. Like, `django.utils` contains all functions and classes that are just general utility classes I guess? So personally I would prefer something like `utils/interfaces` where I can import any interface.

Reply:
ok